src/server/protocol.ts

```diff
@@ -1541,6 +1561,7 @@ namespace ts.server.protocol {
         description: string;
         /** Text changes to apply to each file as part of the code action */
         changes: FileCodeEdits[];
+        commands?: {}[];

```

Comment:
we should add a note that this is an opaque object to be passed as the input to the apply command request.